docs/README.md

```diff
@@ -0,0 +1,4 @@
+
+Detailed documentation per model
+
+* [Regressor model for predicting risky commits](models/regressor.md)
```

Comment:
Could you add the same link in the main README (in the https://github.com/mozilla/bugbug#classifiers section).

Reply:
done